### PR TITLE
Fix a typo in ApiCallSettings snip

### PR DESCRIPTION
**Pull request by shinfan**

### CODE REVIEW

File: src/test/java/com/google/api/codegen/testdata/java_settings_library.baseline [hunk 1]
```diff
@@ -717,9 +717,6 @@ public class LibraryServiceSettings extends ServiceApiSettings {
           addCommentsSettings,
           getBookFromArchiveSettings,
           updateBookIndexSettings,
-          streamShelvesSettings,
```
Comment (michaelbausor) [inline src/test/java/com/google/api/codegen/testdata/java_settings_library.baseline]: Why are these disappearing as part of this change?

Comment (shinfan) [inline src/test/java/com/google/api/codegen/testdata/java_settings_library.baseline]: Those settings are streaming settings which should not be a part of unaryMethodSettingsBuilders array.

Comment (michaelbausor) [inline src/test/java/com/google/api/codegen/testdata/java_settings_library.baseline]: Yes, but what change in toolkit is making them disappear? So, how is xsettingsClass.unaryCallSettings changing?

Comment (shinfan) [inline src/test/java/com/google/api/codegen/testdata/java_settings_library.baseline]: ummm before the template was referring  xsettingsClass.callSettings which contains all settings but I corrected it to xsettingsClass.unaryCallSettings which only contains unary settings. Does it answer you question?

**Comment by garrettjonesgoogle** [comment]
LGTM

**Pull request merged**